docs/nginx-ingress-controller.md

```diff
@@ -147,6 +147,7 @@ spec:
 | --- | --- | --- | --- |
 | `enable` | `boolean` | Enable Prometheus metrics. | Yes |
 | `port` | `int` | Sets the port where the Prometheus metrics are exposed. Default is 9113. Format is `1023 - 65535`. | No |
+| `secret` | `string` | Sets the namespace/name of a TLS Secret Resource to use to enable TLS for the Prometheus endpoint. | No |
```

Comment:
think need to add arg above line 76 too

Reply:
I've updated the example if that's what you mean? I don't see anywhere else to add it

Reply:
yep that was it